Python/ceval.c

```diff
@@ -3918,8 +3951,14 @@ _PyEval_EvalFrameDefault(PyThreadState *tstate, _PyInterpreterFrame *frame, int
             JUMP_TO_INSTRUCTION(JUMP_BACKWARD_QUICK);
         }
 
-        TARGET(POP_JUMP_IF_FALSE) {
-            PREDICTED(POP_JUMP_IF_FALSE);
+        TARGET(POP_JUMP_BACKWARD_IF_FALSE) {
+            PREDICTED(POP_JUMP_BACKWARD_IF_FALSE);
+            oparg = -oparg;
+            JUMP_TO_INSTRUCTION(POP_JUMP_FORWARD_IF_FALSE);
```

Comment:
We probably want to change this like in the other PR.

Reply:
Yes